src/targets/cpu/lowering.cpp

```diff
@@ -398,6 +400,10 @@ struct cpu_apply
             {
                 apply_pooling(it);
             }
+            else if(it->name() == "convolution_backwards" and enabled(MIGRAPHX_ENABLE_ZENDNN{}))
```

Comment:
There is already the macro `MIGRAPHX_ENABLE_ZENDNN` that gets defined when we are using zendnn. This code also probably breaks when that macro gets defined, so you should be using `#ifdef MIGRAPHX_ENABLE_ZENDNN` instead.